docs/v3_services.rst

```diff
@@ -1,61 +1,55 @@
 SecureDrop V3 Onion Services
 ============================
-Tor onion services provide anonymous inbound connections to websites and other
-servers exclusively over the Tor network. SecureDrop uses onion services 
-for the *Journalist* and *Source Interface* websites, as well as for 
-adminstrative access to the servers in SSH-over-Tor mode. 
 
-Currently, SecureDrop supports two versions of the onion services
-protocol: the older version (v2), and the current version (v3), which provides 
-stronger cryptography and includes redesigned protocols that guard against 
-service information leaks on the Tor network. Because of these important 
-improvements, the Tor project is
-`deprecating support for v2 onion services <https://blog.torproject.org/v2-deprecation-timeline>`__. 
+.. important::
 
-SecureDrop will begin phasing out support for v2 onion services in 
-**February 2021**. All newsrooms should enable v3 onion services as soon as
-possible, but may continue using v2 concurrently to allow all journalists and 
-sources to transition smoothly.
+   If you have not enabled v3 onion services before April 30, 2021,
+   your SecureDrop servers will become unreachable. To minimize disruption
+   to sources and journalists, you should begin the migration well
+   before that deadline.
 
-.. warning::
+   Your SecureDrop's :ref:`Landing Page <glossary_landing_page>` 
+   must be updated with your new 56-character onion address so sources 
+   can continue to reach you.
 
-   If you are still using 16-character v2 onion URLs in April 2021, your SecureDrop 
-   servers will become unreachable, and you will have to reinstall SecureDrop 
-   from scratch.
+SecureDrop supports the v3 :ref:`onion services <glossary_onion_service>`
+protocol, which provides stronger cryptographic protections than v2 onion 
+services, and helps mitigate service information leaks on the Tor network. 
+Because of these important improvements, the Tor project is 
+`deprecating support for v2 onion services <https://blog.torproject.org/v2-deprecation-timeline>`__,
+and SecureDrop is also phasing out support for the v2 protocol. 
 
-How to distinguish between v2 and v3 onion services
----------------------------------------------------
+SecureDrop administrators should enable v3 onion services as soon as possible.
+This action will not affect sources or journalists, since the v2 onion 
+URLs will continue to work. Then, administrators should ensure that the
```

Comment:
> This action will not affect sources or journalists, since the v2 onion 
URLs will continue to work 

I feel like we need to say something like this so that administrators are aware they can take the first migration step without having to coordinate with journalists.

Reply:
Makes sense, I just tweaked this slightly to avoid misinterpretation of "will continue to work".